deduplicator/src/bin/main.rs

```diff
@@ -56,6 +61,14 @@ struct Params {
     /// Number of pages to be used by SQLite (one page is 4096 bytes)
     #[structopt(short, long, default_value = "10000")]
     cache_size: u32,
+
+    /// Number of thread to target during the computation.
+    #[structopt(short, long)]
+    num_threads: Option<usize>,
+
+    /// Redraw delay for displayed progress (in ms)
+    #[structopt(long, default_value = "1000", parse(try_from_str = parse_duration))]
+    refresh_delay: Duration,
```

Comment:
This default value is too low for production use. 
So a PR will be needed to override this value in docker_mimir.

Reply:
Indeed, I only ran doc tests locally ...

I'll open a PR on docker_mimir as soon as this one is merged :)